packages/components/src/form-token-field/test/index.tsx

```diff
@@ -0,0 +1,2089 @@
+				screen.getByRole( 'combobox' )
+			).toHaveAccessibleDescription( instructionsTokenizeSpace );
+
+			rerender(
+				<FormTokenFieldWithState
+					tokenizeOnSpace={ true }
+					__experimentalShowHowTo={ false }
+				/>
+			);
+
+			expect(
+				screen.queryByText( instructionsDefault )
+			).not.toBeInTheDocument();
+			expect(
+				screen.queryByText( instructionsTokenizeSpace )
+			).not.toBeInTheDocument();
+			expect(
+				screen.getByRole( 'combobox' )
+			).not.toHaveAccessibleDescription();
+		} );
+
+		it( "should use the value of the `placeholder` prop as the input's placeholder only when there are no tokens", async () => {
+			const user = userEvent.setup( {
+				advanceTimers: jest.advanceTimersByTime,
+			} );
+
+			const onChangeSpy = jest.fn();
+
+			render(
+				<FormTokenFieldWithState
+					onChange={ onChangeSpy }
+					placeholder="Test placeholder"
+				/>
+			);
+
+			expect(
+				screen.getByPlaceholderText( 'Test placeholder' )
+			).toBeVisible();
+
+			const input = screen.getByRole( 'combobox' );
+
+			// Add 'blueberry' token. The placeholder text should not be shown anymore
+			await user.type( input, 'blueberry[Enter]' );
+			expect( onChangeSpy ).toHaveBeenCalledTimes( 1 );
+			expect( onChangeSpy ).toHaveBeenCalledWith( [ 'blueberry' ] );
+			expectTokensToBeInTheDocument( [ 'blueberry' ] );
+
+			expect(
+				screen.queryByPlaceholderText( 'Test placeholder' )
+			).not.toBeInTheDocument();
+		} );
+	} );
+
+	describe( 'suggestions', () => {
+		it( 'should not render suggestions in its default state', () => {
+			render(
+				<FormTokenFieldWithState
+					suggestions={ [ 'Red', 'Magenta', 'Vermilion' ] }
+				/>
+			);
+
+			expect( screen.queryByRole( 'listbox' ) ).not.toBeInTheDocument();
+		} );
+
+		it( 'should render suggestions when receiving focus if the `__experimentalExpandOnFocus` prop is set to `true`', async () => {
+			const user = userEvent.setup( {
+				advanceTimers: jest.advanceTimersByTime,
+			} );
+
+			const onFocusSpy = jest.fn();
+
+			const suggestions = [ 'Cobalt', 'Blue', 'Octane' ];
+
+			render(
+				<>
+					<FormTokenFieldWithState
+						onFocus={ onFocusSpy }
+						suggestions={ suggestions }
+						__experimentalExpandOnFocus={ true }
+					/>
+				</>
+			);
+
+			const input = screen.getByRole( 'combobox' );
+
+			expect( screen.queryByRole( 'listbox' ) ).not.toBeInTheDocument();
+
+			// Click the input, focusing it.
+			await user.click( input );
+
+			const suggestionList = screen.getByRole( 'listbox' );
+
+			expect( onFocusSpy ).toHaveBeenCalledTimes( 1 );
+			expect( suggestionList ).toBeVisible();
+
+			expect(
+				within( suggestionList ).getAllByRole( 'option' )
+			).toHaveLength( suggestions.length );
+
+			// Minimum length limitations don't affect the search text when the
+			// `__experimentalExpandOnFocus` is `true`
+			await user.keyboard( 'c' );
+			expectVisibleSuggestionsToBe( screen.getByRole( 'listbox' ), [
+				'Cobalt',
+				'Octane',
+			] );
```

Comment:
Could we use `expectVisibleSuggestionsToBe` here?

Reply:
Done in https://github.com/WordPress/gutenberg/pull/43442/commits/46d35d5560450b17794631e5622bbe6e338bb334